src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final int SUPPLY_AMOUNT = 0;
+    private static final int BUY_AMOUNT = 1;
+    private static final int RESULT_AMOUNT = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int [] report = createReport(fromFileName);
+        writeToFile(report,toFileName);
+    }
+
+    private int[] createReport(String fromFileName) {
```

Comment:
this method should be divided into two (one for reading from a file and the second one for forming of report)